internal/monitor/features/controller.feature

```diff
@@ -25,6 +25,20 @@ Feature: Controller Monitor
       | "node1" | 2    | "ValidateVolumeHostConnectivity" | "node1" | "false"   | "Aborting pod cleanup due to error"                  |
       | "node1" | 2    | "CreateEvent"                    | "node1" | "true"    | "Successfully cleaned up pod"                        |
 
+
+   @controller-mode
+  Scenario Outline: Test controllerCleanupPodWithRWX
+    Given a controller monitor "vxflex"
```

Comment:
Is this only supported for powerflex?

Reply:
This is supported for all drivers. I have updated the test to test for all drivers.